applications/FemToDemApplication/custom_problemtype/FemDemKratos.gid/MainKratos.py

```diff
@@ -1,7 +1,7 @@
-import KratosMultiphysics
-import KratosMultiphysics.SolidMechanicsApplication
-import KratosMultiphysics.FemToDemApplication
-import CouplingFemDem
-
-model = KratosMultiphysics.Model()
-CouplingFemDem.FEMDEM_Solution(model).Run()
\ No newline at end of file
+import KratosMultiphysics
+import KratosMultiphysics.SolidMechanicsApplication
+import KratosMultiphysics.FemToDemApplication
+import MainCouplingFemDem
```

Comment:
be careful, this type of importing the python scripts will stop working very soon => #5268

Reply:
Okkey @philbucher , in the next days I'll try to update it. THX for the info

Reply:
:+1: 
You can check StructuralMechanics for a reference, there the imports are already updated